<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuiZip</class>
 <widget class="QDialog" name="GuiZip">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Archivos Comprimidos</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recursos.qrc">
    <normaloff>:/img/iconos/portada.png</normaloff>:/img/iconos/portada.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>3.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">QProgressBar {
    border: 2px solid grey;
    border-radius: 5px;
	text-align: center;
}

 QProgressBar::chunk {
     background-color: #05B8CC;
     width: 20px;

 }


</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Comentarios</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2" colspan="4">
    <widget class="QTextEdit" name="txtComentarios">
     <property name="maximumSize">
      <size>
       <width>501</width>
       <height>50</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="btnAbrir">
     <property name="text">
      <string>Abrir</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="5">
    <widget class="QTreeWidget" name="treeWidget">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustIgnored</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="animated">
      <bool>true</bool>
     </property>
     <property name="allColumnsShowFocus">
      <bool>true</bool>
     </property>
     <property name="columnCount">
      <number>4</number>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">2</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">3</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">4</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="btnDescomprimir">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Descomprimir todo</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="btnDescomprimirIndividual">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Descomprimir 1 archivo</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Archivos</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QProgressBar" name="progressBar">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QPushButton" name="brnDetener">
     <property name="text">
      <string>detener</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="btnvisualizar">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>visualizar</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
